Core/Logout.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Logout.swift
+//  AuthorizeMe
+//
+//  Created by Radislav Crechet on 6/20/17.
+//  Copyright © 2017 RubyGarage. All rights reserved.
+//
+
+import Foundation
+
+public class Logout {
+    
+    public static let me = Logout()
+    
+    private var provider: Provider!
+    
+    public func from(_ nameOfProvider: String) {
+        let bundleName = Bundle(for: type(of: self)).infoDictionary!["CFBundleName"] as! String
+        
+        guard let provider = NSClassFromString("\(bundleName).\(nameOfProvider)Provider") as? Provider.Type else {
```

Comment:
I think we need to extract converting string to Provider.Type to separate class. Because the similar functionality already exists in https://github.com/rubygarage/authorize-me/blob/master/Core/Authorize.swift#L21